src/components/counters/Counter.jsx

```diff
@@ -0,0 +1,144 @@
+// @flow strict
+
+import React from 'react';
+import cx from 'classnames';
+import Text from '../text/Text';
+import Flex from '../flex/Flex';
+import Icon from '../icons/Icon';
+
+type CounterTypeType = 'text' | 'points';
+type CounterSizeType = 'normal' | 'small';
+
+export const COUNTER_TYPE = Object.freeze({
+  BASIC: 'basic',
+  POINTS: 'points',
+});
+
+export const COUNTER_SIZE = Object.freeze({
+  NORMAL: 'normal',
+  SMALL: 'small',
+});
+
+export type CounterPropsType = {
+  /**
+   * Children to be rendered inside Counter
+   * @example <Counter type="basic">
+   *            text
+   *          </Counter>
+   */
+  children?: Node,
+  /**
+   * Specify type of the counter that you want to use, to types for now
```

Comment:
did you mean '...two types for now'? :)

Reply:
@ElaLukaszewicz yeah, sorry. on my way to fix it.